custom-connectors/Zenkraft/Readme.md

```diff
@@ -0,0 +1,27 @@
+The Zenkraft shipping API is designed to make it easy to rate, book, track and schedule pickups with a growing list of shipping carriers.
+
+## How to get credentials
+You will need a paid subscription to access the Zenkraft API. Please contact sales@zenkraft.com
+
+
+## Prerequisites
+You will need the following to proceed:
+* A Zenkraft API Key
+
+## Known issues and limitations
+If there's an issue with a carrier's web service, our connector will return an error message
+
+
+## Supported Operations
+The connector supports the following operations:
+* `Get rates
+* `Create Shipment
+* `Pickup Shipment
+* `Cancel Shipment`
+* `Track Shipment`
+* `Print Label
+* `Get Printers
```

Comment:
It looks like the formatting is inconsistent here. They should all be within backticks, like `Cancel Shipment` and `Track Shipment` are.